### PR TITLE
feat: csv support lazy quotes

### LINKED ISSUE #221: Failed to parse double quotes, and failed records were not logged to the error file
image

image

Consider more special characters when solving。

### PR DESCRIPTION
**Pull request by veezhang**
feat: csv support lazy quotes

## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Issue(s) number: 

fix #221

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +173,7 @@ One CSV file can only store one type of vertex or edge. Vertices and edges of th
   * `withHeader`: The default value is false. The format of the header is described in the following section.
   * `withLabel`: The default value is false. The format of the label is described in the following section.
   * `delimiter`: **Optional**. Specify the delimiter for the CSV files. The default value is `","`. And only a 1-character string delimiter is supported.
+  * `lazyQuotes`: **Optional**. If `lazyQuotes` is true, a quote may appear in an unquoted field and a non-doubled quote may appear in a quoted field.
```
Comment (yixinglu) [inline README.md:176]: you'd better to do some checks whether the delimiter is quote.

Comment (veezhang) [reply README.md:176]: It's necessary? The default value is `false`.

Comment (veezhang) [reply README.md:176]: BTW. the `"` is not a valid delimiter.

https://github.com/golang/go/blob/master/src/encoding/csv/reader.go#L97-L99

Comment (yixinglu) [reply README.md:176]: have no question!

**Review by yixinglu — APPROVED** [review]

**Pull request merged**